docs/devel/testing.md

```diff
@@ -98,18 +98,23 @@ You can set [go flags](https://golang.org/cmd/go/) by setting the
 
 ### Run unit tests from certain packages
 
-The `hack/test-go.sh` script accepts packages as arguments; the
-`k8s.io/kubernetes` prefix is added automatically to these:
+`make test` accepts packages as arguments; the `k8s.io/kubernetes` prefix is
+added automatically to these:
 
 ```sh
-hack/test-go.sh pkg/api             # run tests for pkg/api
-hack/test-go.sh pkg/api pkg/kubelet # run tests for pkg/api and pkg/kubelet
+make test WHAT=pkg/api                # run tests for pkg/api
+```
+
+To run multiple targets you need quotes:
+
+```sh
+make test WHAT="pkg/api pkg/kubelet"  # run tests for pkg/api and pkg/kubelet
```

Comment:
Not automatically.  It is exactly the same as hack/build-go.sh.  ./foo/... does not seem to work (it should, I think, but it never did)